### ISSUE TITLE
Create a vote dialog

### ISSUE THREAD

**Issue by PeterMPhillips** [coeo/coeo-client#40]
Make a dialog for voting instead of voting directly on the Decision tile
![paste (4)](https://user-images.githubusercontent.com/19808076/89852949-e7b6d700-db44-11ea-86c4-b64042f48ec6.png)

**Comment by stellarmagnet** [comment]
I think I have to think through this design more since what you vote will
be anonymous but the comment won’t be.

**Comment by PeterMPhillips** [comment]
good point. prolly best to keep those things separate. seems fine to just remove the comment box, no? also, since we're doing 'x' in the corner, we should just have a full width 'Submit' button
